.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-name: .NET 5.0
+name: .NET 6.0
```

Comment:
this would require a change in the ReadMe too, see: https://github.com/fsprojects/FSharpLint/commit/861d10e3ed4f5388f0c65e0e1bc0f49e8f0a4246